botx/clients/methods/errors/no_permission.py

```diff
@@ -0,0 +1,51 @@
+"""Definition for "no permission" error."""
+from typing import NoReturn
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from botx.clients.methods.base import APIErrorResponse, BotXMethod
+from botx.clients.types.http import HTTPResponse
+from botx.exceptions import BotXAPIError
+
+
+class NoPermissionError(BotXAPIError):
+    """Error for raising when there is no permission for operation."""
+
+    message_template = (
+        "Bot doesn't have permission for this operation in chat {group_chat_id}"
+    )
+
+    #: ID of chat that was requested.
+    group_chat_id: UUID
+
+
+class NoPermissionErrorData(BaseModel):
+    """Data for error when there is no permission for operation."""
+
+    #: ID of chat that was requested.
+    group_chat_id: UUID
+
+
+def handle_error(method: BotXMethod, response: HTTPResponse) -> NoReturn:
```

Comment:
You use `None` as annotation in all other places, and `None` as no-return annotation used in all project

Reply:
NoReturn is used in all modules with errors in  botx.clients.methods.errors